plugins/locator/nll/nll_locator.cpp

```diff
@@ -297,6 +298,23 @@ double normalizeLon(double lon) {
 
 
 
+// >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+std::string stationName(Seiscomp::DataModel::Pick *pick, const std::string& rule)
+{
+  static const std::regex reNet("NETCODE", std::regex::optimize);
```

Comment:
Actually we have `Utils::replace` which can be used to replace e.g.: "@NET@.@STA@.@LOC@". The prefix and postfix can be set to anything you want.

Reply:
Good to know. I'll try using `Utils::replace` then